app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Show Recipes </h1>
+
+<div class="recipe-details">
+    <h2 class="recipe-name"> <%= @recipes.name %> </h2>
+  <div class="recipe-status">
+    
+    <div class="left-block">
+      <span>Preparation time : <%= @recipes.preparation_time%> </span>
+      <p>Cooking time: <%=  @recipes.cooking_time %> hours</p>
+    </div>
+    <div class="right-block">
+      <span>Public: <%= check_box :public, value: "true"%></span> 
+    </div>
+ </div>
+  <h2> Steps to prepare </h2>
+
+<div class="recipe-actions">
+    <%= link_to 'Generate shopping list', class:"recipe-btn"%>
```

Comment:
- [ Required ] Please consider making the `Generate shopping list` button work, currently, when I click on the link nothing happens. The project requires that when it is clicked, it should open a shopping list page that matches the wireframe shown below.

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/general_shopping_list.png?token=GHSAT0AAAAAABTYBUDZPKGNYUE7BYLQRJOMY7QZQ7A)

Reply:
Okay. Thank you very much